core/src/main/resources/hudson/model/ListView/configure-entries.jelly

```diff
@@ -30,6 +30,7 @@ THE SOFTWARE.
  <f:section title="${%Job Filters}">
   
   <f:entry field="recurse">
+    <st:adjunct includes="hudson.model.ListView.listView" />
```

Comment:
💡 I would propose you to move this to the top, just below the `j:jelly` to help readability. Putting it anywhere in the page reduce the likelihood the reader will discover it easily

Reply:
Fixed